src/bin/make_data.js

```diff
@@ -9,11 +9,15 @@ import path from 'path';
 
 import * as _ from 'lamb';
 import yaml from 'js-yaml';
-import {readDir, readFile, saveObj} from '@svizzle/file';
+import {readDir, readFile, saveObj, saveString} from '@svizzle/file';
 import {tapMessage} from '@svizzle/dev';
 import {applyFnMap} from '@svizzle/utils';
 
-import {indexById, makeDatasetBySource} from 'app/utils';
+import {indexById, makeDatasetBySource, generateTypingsForAll} from 'app/utils';
+
+const ESTYPINGS_PATH = path.resolve(__dirname, '../node_modules/app/elasticsearch/typings/es.ts');
+const DSLTYPINGS_PATH = path.resolve(__dirname, '../node_modules/app/elasticsearch/typings/dsl.ts');
+const TYPINGS_PATH = path.resolve(__dirname, '../../static/dsl/datasets.ts');
```

Comment:
`static/dsl/datasets.ts` doesn't seem to be resolved, I guess this needs to be added?